Sources/SwiftModCore/Extensions/IndentExtensions.swift

```diff
@@ -7,7 +7,7 @@ public extension Indent {
             return .spaces(spaces)
 
         case .tab:
-            return .tabs(1)
+            return .tab
```

Comment:
Note: Replacing with shorter trivia syntaxes like `.tab` / `.space` / etc. in this PR.